modules/operations/pages/monitoring/new-relic.adoc

```diff
@@ -1,67 +1,69 @@
-= New Relic Integration
+= Integrate with New Relic
 
-[NOTE]
-====
-This article is a continuation of xref:monitoring/index.adoc[]. Please read that article first to understand the fundamentals of what resources are being used.
-====
+While there are multiple ways to https://docs.newrelic.com/docs/infrastructure/prometheus-integrations/get-started/send-prometheus-metric-data-new-relic[send external Prometheus data to New Relic], one the https://docs.newrelic.com/docs/infrastructure/prometheus-integrations/get-started/send-prometheus-metric-data-new-relic/#remote-write[Prometheus remote write integration] option is relevant to {product}.
 
-According to the https://docs.newrelic.com/[New Relic documentation], there are 3 different ways to integrate external Prometheus data into New Relic:
+== Prerequisites
 
-* Option 1: https://docs.newrelic.com/docs/infrastructure/prometheus-integrations/get-started/send-prometheus-metric-data-new-relic/#Agent[Prometheus Agent for Kubernetes]
-* Option 2: https://docs.newrelic.com/docs/infrastructure/prometheus-integrations/get-started/send-prometheus-metric-data-new-relic/#OpenMetrics[Prometheus OpenMetrics integration for Docker]
-* Option 3: https://docs.newrelic.com/docs/infrastructure/prometheus-integrations/get-started/send-prometheus-metric-data-new-relic/#remote-write[Prometheus remote write integration]
+* Review xref:monitoring/index.adoc[].
+* https://docs.newrelic.com/[Set up a New Relic account.]
+* Save your New Relic license key locally.
```

Comment:
Looks like the preview is out of date. The .adoc file has the correct capitalization

![new-relic](https://github.com/user-attachments/assets/dce2c826-42c6-4072-8542-0ac07741d126)